# Calibration parameters
# Amount of readings used to average, make it higher to get more precision but
# sketch will be slower (default:1000)
buffer_size: 1000
# Accelerometer error allowed, make it lower to get more precision, but sketch
# may not converge (default:8)
accel_deadzone: 8
# Gyro error allowed, make it lower to get more precision, but sketch may not
# converge (default:1)
gyro_deadzone: 1
